### PR TITLE
feat(material-requests): add functionality to retreive material reque…

### PR DESCRIPTION
**Pull request by tourlbr**
Note: gebruik maken van feat/ARC-1200 in hetarchief-hasura bij het testen

Screenshot 2023-01-17 at 17 00 57

### CODE REVIEW

**Review by WardVerc1 — APPROVED** [review]

File: src/modules/material-requests/controllers/material-requests.controller.ts [hunk 1]
```diff
@@ -36,4 +37,13 @@ export class MaterialRequestsController {
 
 		return visits;
 	}
+
+	@Get(':id')
+	@RequireAnyPermissions(
+		Permission.READ_ALL_MATERIAL_REQUESTS,
```
Comment (bertyhell) [inline src/modules/material-requests/controllers/material-requests.controller.ts:43]: Rename to better match the naming convention for the other permissions
```suggestion
		Permission.VIEW_ANY_MATERIAL_REQUESTS,
```

File: src/modules/material-requests/queries/find-by-id-material-requests.graphql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+query findMaterialRequestsById($id: uuid!) {
+	app_material_requests(where: { id: { _eq: $id } }) {
+		id
+		object_schema_identifier
+		profile_id
+		reason
+		created_at
+		updated_at
```
Comment (bertyhell) [inline src/modules/material-requests/queries/find-by-id-material-requests.graphql:8]: We'll need more than just this info to render the detail view:
image

- maintainerName
- maintainerLogo
- objectName
- objectPid
- requesterName
- requesterEmail
- RequesterUserGroup

**Review by bertyhell — CHANGES_REQUESTED** [review]

File: src/modules/material-requests/controllers/material-requests.controller.ts (hunk 1 above)
Comment (tourlbr) [reply src/modules/material-requests/controllers/material-requests.controller.ts:38]: Adjusted

File: src/modules/material-requests/queries/find-by-id-material-requests.graphql (hunk 2 above)
Comment (tourlbr) [reply src/modules/material-requests/queries/find-by-id-material-requests.graphql:8]: Adjusted

File: src/modules/users/types.ts [hunk 3]
```diff
@@ -50,8 +50,8 @@ export enum Permission {
 	// IDP-based permission:
 	CAN_EDIT_PROFILE_INFO = 'CAN_EDIT_PROFILE_INFO',
 	// Material Requests
-	READ_PERSONAL_APPROVED_MATERIAL_REQUESTS = 'READ_PERSONAL_APPROVED_MATERIAL_REQUESTS',
 	READ_ALL_MATERIAL_REQUESTS = 'READ_ALL_MATERIAL_REQUESTS',
+	VIEW_OWN_MATERIAL_REQUESTS = 'VIEW_OWN_MATERIAL_REQUESTS',
```
Comment (bertyhell) [inline src/modules/users/types.ts:54]:
```suggestion
	VIEW_ANY_MATERIAL_REQUESTS = 'VIEW_ANY_MATERIAL_REQUESTS',
	VIEW_OWN_MATERIAL_REQUESTS = 'VIEW_OWN_MATERIAL_REQUESTS',
```

**Review by bertyhell — CHANGES_REQUESTED** [review]

**Pull request merged**